src/configManager.ts

```diff
@@ -0,0 +1,93 @@
+import { ProductDetails } from './product';
+import { SdkOptions } from './sdkOptions';
+
+type storedConfig = Pick<
+  SdkOptions,
+  'baseUrl' | 'useCookie' | 'refreshRetryPeriod' | 'cookiePath' | 'cookieDomain'
+>;
+
+const getConfigFromSdkOptions = (options: SdkOptions): storedConfig => {
+  const config: storedConfig = {
+    refreshRetryPeriod: options.refreshRetryPeriod,
+    baseUrl: options.baseUrl,
+    useCookie: options.useCookie,
+    cookiePath: options.cookiePath,
+    cookieDomain: options.cookieDomain,
+  };
+  return config;
+};
+
+export const storeConfig = (options: SdkOptions, productDetails: ProductDetails) => {
+  if (options.useCookie) {
+    setConfigCookie(options, productDetails);
+  } else {
+    setConfigToLocalStorage(options, productDetails);
+  }
+};
+
+export const loadConfig = (
+  options: SdkOptions,
+  productDetails: ProductDetails
+): storedConfig | null => {
+  if (options.useCookie) {
+    return loadConfigFromCookie(productDetails);
+  } else {
+    return loadConfigFromLocalStorage(productDetails);
+  }
+};
+
+const setConfigCookie = (options: SdkOptions, productDetails: ProductDetails) => {
+  const cookieDomain = options.cookieDomain;
+  const path = options.cookiePath ?? '/';
+  const value = JSON.stringify(getConfigFromSdkOptions(options));
+  let cookie =
+    productDetails.cookieName + '_config' + '=' + encodeURIComponent(value) + ' ;path=' + path;
+  if (typeof cookieDomain !== 'undefined') {
+    cookie += ';domain=' + cookieDomain;
+  }
+  document.cookie = cookie;
+};
```

Comment:
could you call the setCookie() function here since it does the same thing?

Reply:
setCookie() specifically sets an Identity cookie.  I didn't want to overcomplicate that function so I made a new, specific one.